src/docs/forms/components/CalendarInput/examples/05-isSelectableDate.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import moment from 'moment';
+import {CalendarInput} from "doj-react-adminlte/Form";
+
+
+export default class isSelectableDate extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            Calendarone: null,
+            Calendartwo: null
+        };
+    }
+
+    isValidDate = current => {
+        const yesterday = moment().subtract(1, 'day');
+        return current.isAfter(yesterday);
+    };
+
+    handleDateChange = (name, value) => {
+        this.setState({[name]: value});
+    };
+
+    render() {
+        const {Calendarone,Calendartwo} = this.state;
+
+        return (
+            <div className="row">
+                <div className="col-xs-6">
+                    <CalendarInput name="Calendarone" label="Disabled Calendar Input" value={Calendarone}
+                                   onChange={this.handleDateChange}  isSelectableDate={this.isValidDate}/>
+                </div>
+                <div className="col-xs-6">
+                    <CalendarInput name="Calendartwo" label="Calendar Input" value={Calendartwo}
```

Comment:
Keep your first example, then instead of this, create another example that will disable all weekends, another example that will disable all future dates, and another example that will disable all wednesdays.